src/libs/ReportActionsUtils.ts

```diff
@@ -80,6 +80,10 @@ function isMoneyRequestAction(reportAction: OnyxEntry<ReportAction>): boolean {
     return reportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.IOU;
 }
 
+function isDoneAction(reportAction: OnyxEntry<ReportAction>): boolean {
+    return reportAction?.actionName === CONST.REPORT.ACTIONS.TYPE.CLOSED;
```

Comment:
Full disclosure I did not read the design doc related to this change. The CLOSED action also used to indicate an "archived" chat. So, maybe we should also check the report to see what kind it is? I am lacking the full context though about why we decided to reuse this action to mean both "done" or "archived".